docs/card.md

```diff
@@ -214,6 +218,27 @@ specify styling for view surrounding image
 | :-----------: | :-----: |
 | object(style) |  none   |
 
+
+---
+
+### `imageOnPress`
+
+trigger header image press (optional)
+
+|   Type   | Default |
+| :------: | :-----: |
+| function |  none   |
+
+---
+
+### `imageOnPressProps`
+
+optional properties to pass to the TouchableHighlight element generated by imageOnPress e.g "underlayColor"
+
+|                 Type                 | Default |
+| :----------------------------------: | :-----: |
+| object(TouchableHighlightProperties) |  none   |
```

Comment:
Use the format from there to describe the type https://github.com/react-native-training/react-native-elements/blame/master/docs/listitem.md#L328

`{[...TouchableHighlight props](https://facebook.github.io/react-native/docs/touchablehighlight#props)}`

Reply:
did it